stack.bat

```diff
@@ -0,0 +1,25 @@
+@echo on
+SETLOCAL
+
+set CMD=%1
+CALL :%CMD%
+
+:Up
+docker volume prune -f
+docker compose -p service-web-store-env -f docker/dependencies.yaml up --build -d
+docker ps -a
+EXIT /B 0
+
+:Down
+docker compose -p service-web-store-env -f docker/dependencies.yaml down
+EXIT /B 0
+
+:Bounce
+CALL :Down
+CALL :Up
+EXIT /B 0
+
+:Reset
+CALL :Down
+rm -rf ./docker/data
+EXIT /B 0
```

Comment:
My Windows batch fu is super out dated and I don't have a windows machine. This may not work at all. But the way it should work is, from the console you can run the batch file and pass either "Up", "Down", "Bounce" or "Reset" as the first arg, which will call the corresponding function.

Up - Starts the docker services in the dependencies file
Down - Stops the docker services
Bounce - Restarts the docker services
Reset - Stops the docker services and purges the persistent data volume